.factorypath

```diff
@@ -0,0 +1,34 @@
+<factorypath>
```

Comment:
this file should not be committed to git - its a mvn and machine specific

Reply:
I've already included in ,gitignore in a previous [PR](https://github.com/maxandersen/starfish/pull/14) that is yet to be merged

Reply:
okey, but just do `git rm .factorypath` and update the PR then it doesn't get added.